scripts/angry-tock.js

```diff
@@ -218,8 +218,8 @@ let shout = robot => {
         });
       } else {
         robot.messageRoom('18f-gmt', {
-          username: 'Happy Tock',
-          icon_emoji: ':happy-tock:',
+          username: 'Disappointed Tock',
+          icon_emoji: ':disappointed-tock:',
           text: 'No Tock truants!',
           as_user: false
```

Comment:
This is for the case where there aren't any truants. That doesn't seem like when we'd want to use disappointed Tock?

Reply:
Derp.